src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const styleObject = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `styleObject`. It's clear from the context that this is an object related to styles, so a more descriptive name might be `styles` or `cssStyles`.